### PR TITLE
Fix: wrong range of histogram in drop_pixel

### PR DESCRIPTION
**Pull request by narduzzi**

### CODE REVIEW

File: tonic/functional/to_frame.py [hunk 1]
```diff
@@ -66,9 +66,12 @@ def to_frame_numpy(
         events["p"] = 0
 
     if time_window:
-        event_slices = slice_events_by_time(
-            events, time_window, overlap=overlap, include_incomplete=include_incomplete
-        )
+        if len(events) == 0:
```
Comment (biphasic) [inline tonic/functional/to_frame.py:69]: the check for the case that an empty list of events is passed should probably happen a bit higher up, for example here:
https://github.com/neuromorphs/tonic/blob/7455d0084710c95a4a6d56eed7f0c07cfaccd238/tonic/transforms.py#L894

File: AUTHORS [hunk 2]
```diff
@@ -0,0 +1,45 @@
+Anindya Ghosh <anindya128@yahoo.com>
+Antoine Grimaldi <antoine.grimaldi@univ-amu.fr>
+AntoineGrimaldi <antoine.grimaldi@univ-amu.fr>
+Carsten Nielsen <carstenlau.nielsen@uzh.ch>
+Fabrizio Ottati <45739782+fabhertz95@users.noreply.github.com>
+Fabrizio Ottati <fabriziottati@gmail.com>
```
Comment (biphasic) [inline AUTHORS:6]: I'm not sure why this file hasn't been excluded in the gitignore already, could you please remove this file and ChangeLog. I fixed the .gitignore in the current develop branch

File: tonic/functional/drop_pixel.py [hunk 3]
```diff
@@ -17,12 +17,15 @@ def identify_hot_pixel(events: np.ndarray, hot_pixel_frequency: float):
 
     assert "x" and "y" and "t" in events.dtype.names
 
+    if len(events)==0:
+        return []
+
```
Comment (biphasic) [inline tonic/functional/drop_pixel.py:22]: If an emty list is passed to the tonic transform, the functional call shouldn't be called in the first place. The check should probably happen somewhere here 
https://github.com/neuromorphs/tonic/blob/7455d0084710c95a4a6d56eed7f0c07cfaccd238/tonic/transforms.py#L230

File: tonic/functional/uniform_noise.py [hunk 4]
```diff
@@ -12,6 +12,9 @@ def uniform_noise_numpy(events: np.ndarray, sensor_size: Tuple[int, int, int], n
         sensor_size: 3-tuple of integers for x, y, p
         n: the number of noise events added.
     """
+    if len(events)==0:
+       return events
+
```
Comment (biphasic) [inline tonic/functional/uniform_noise.py:17]: Same as the previous comments, check for empty arrays should happen in transforms.py

**Review by biphasic — CHANGES_REQUESTED** [review]
Hey Simon, thanks a lot for your PR! Basically the checks for empty arrays should happen in the transforms.py file and not in the functional calls. If you change that and remove the AUTHORS/ChangeLog files it should be good! Thank you

File: tonic/functional/to_frame.py (hunk 1 above)
Comment (narduzzi) [reply tonic/functional/to_frame.py]: Check is now performed in ToFrame transform. Not sure about the behavior, though. The modified implementation is the one that makes most sense to me.

File: tonic/functional/uniform_noise.py (hunk 4 above)
Comment (narduzzi) [reply tonic/functional/uniform_noise.py]: Solved

File: tonic/functional/drop_pixel.py (hunk 3 above)
Comment (narduzzi) [reply tonic/functional/drop_pixel.py]: Solved

**Review by biphasic — APPROVED** [review]

**Pull request merged**